apps/passport-server/src/services/issuanceService.ts

```diff
@@ -133,57 +132,71 @@ export class IssuanceService {
 
   private async issueEmailOwnershipPCD(
     request: IssuedPCDsRequest
-  ): Promise<SerializedPCD | null> {
+  ): Promise<SerializedPCD[]> {
     const email = await this.getUserEmailFromRequest(request);
 
     if (email == null) {
-      return null;
+      return [];
     }
 
-    // TODO: convert from pretix ticket to {@link ITicketData}
-    const ticketData: ITicketData = {
-      attendeeEmail: email,
-      attendeeName: "Ivan Chub",
-      eventName: "ProgCrypto",
-      ticketName: "GA",
-      ticketId: "5",
-      timestamp: Date.now(),
-    };
-
-    const serializedTicketData = JSON.stringify(ticketData);
-    const stableId = await getHash("issued-ticket-" + ticketData.ticketId);
-
-    const rsaPcd = await RSAPCDPackage.prove({
-      privateKey: {
-        argumentType: ArgumentTypeName.String,
-        value: this.exportedPrivateKey,
-      },
-      signedMessage: {
-        argumentType: ArgumentTypeName.String,
-        value: serializedTicketData,
-      },
-      id: {
-        argumentType: ArgumentTypeName.String,
-        value: undefined,
-      },
-    });
-
-    const rsaTicketPCD = await RSATicketPCDPackage.prove({
-      id: {
-        argumentType: ArgumentTypeName.String,
-        value: stableId,
-      },
-      rsaPCD: {
-        argumentType: ArgumentTypeName.PCD,
-        value: await RSAPCDPackage.serialize(rsaPcd),
-      },
-    });
-
-    const serializedTicketPCD = await RSATicketPCDPackage.serialize(
-      rsaTicketPCD
+    const ticketsDB = await fetchDevconnectPretixTicketsByEmail(
+      this.context.dbPool,
+      email
     );
 
-    return serializedTicketPCD;
+    return await Promise.all(
+      ticketsDB
+        // convert to ITicketData
+        .map((t) => ({
+          ticketId: t.id,
+          eventName: t.event_name,
+          ticketName: t.item_name,
+          timestamp: Date.now(),
+          attendeeEmail: email,
+          attendeeName: t.full_name,
+          isConsumed: t.is_consumed,
+          isDeleted: t.is_deleted,
```

Comment:
given the way you've implemented the queries, this is always going to be `false`